app/reports/forms.py

```diff
@@ -45,11 +50,12 @@ class NewIncidentForm(Form):
     date = DateField('Date', default=datetime.date.today(),
                      validators=[InputRequired()])
 
-    duration = IntegerField('Idling Duration (in minutes)', [
-        InputRequired('Idling duration (in minutes) is required.'),
+    # TODO - add support for h:m:s format
```

Comment:
Looks like that the form is rendered with duration in the h:m:s format so it does need to be in this PR.